flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -147,16 +152,6 @@ under the License.
 						</goals>
 						<configuration>
 							<artifactItems>
-								<!-- For WordCount example data -->
-								<artifactItem>
-									<groupId>org.apache.flink</groupId>
-									<artifactId>flink-examples-batch_${scala.binary.version}</artifactId>
-									<version>${project.version}</version>
-									<type>jar</type>
-									<overWrite>false</overWrite>
-									<outputDirectory>${project.build.directory}/classes</outputDirectory>
-									<includes>org/apache/flink/streaming/examples/wordcount/util/WordCountData.class</includes>
-								</artifactItem>
```

Comment:
Turns out that copying this class overwrites the `wordcount/util` folder and deletes CLI.class